react-components/src/architecture/concrete/pointsOfInterest/types.ts

```diff
@@ -37,8 +38,8 @@ export function createPointsOfInterestPropertiesFromPointAndTitle(
     positionZ: cdfPosition.z,
     scene,
     sceneState: {},
-    name: contents[0],
-    description: contents[1],
+    name: contents[0] as string,
+    description: contents[1] as string,
```

Comment:
Burns in my eyes :sweat_smile:

Reply:
It burns my eyes too 🙈but it will be changed definitely after release